sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/PhoneNumberAsyncClientIntegrationTest.java

```diff
@@ -449,6 +452,29 @@ public void releasePhoneNumbersWithResponse() {
             .verifyComplete();
     }
 
+    @Test()
+    public void beginCreateSearch() {
+        List<String> phonePlanIds = new ArrayList<>();
+        phonePlanIds.add("27b53eec-8ff4-4070-8900-fbeaabfd158a");
+
+        CreateSearchOptions createSearchOptions = new CreateSearchOptions();
+        createSearchOptions
+            .setAreaCode(AREA_CODE_FOR_SEARCH)
+            .setDescription("testsearch20200014")
+            .setDisplayName("testsearch20200014")
+            .setPhonePlanIds(phonePlanIds)
+            .setQuantity(2);
+
+        Duration duration = Duration.ofSeconds(5);
+        PhoneNumberAsyncClient client = this.getClient();
+        PollerFlux<PhoneNumberSearch, PhoneNumberSearch> res = client.beginCreateSearch(createSearchOptions, duration);
+        SyncPoller<PhoneNumberSearch, PhoneNumberSearch> sync = res.getSyncPoller();
+        sync.waitForCompletion();
+        PhoneNumberSearch testResult = sync.getFinalResult();
```

Comment:
This checks one path of functionality.. (Create and Validate Response).. but shouldn't we also test that the developer can poll and get the status? Another scenario would be the cancel itself?